### ISSUE TITLE
bug: ion toggle background not updating on IOS

### ISSUE THREAD

**Issue by chrisgriffith** [ionic-team/ionic#19168]
# Ion Toggle background not updating on IOS 

**Ionic version:**
[x] **4.7.4**

**Current behavior:**
The background for the ion-toggle does not pick up the --background color variable on iOS (via Ionic Lab). This value is properly applied for Android.

**Expected behavior:**
The background of the ion-toggle should be the color defined by the --background color variable.

**Steps to reproduce:**
Have an ion-toggle, set the color for the two parts (background and handle) and the two states. 

**Related code:**
```
ion-toggle {
    --background: blue;
    --background-checked: red;
    --handle-background: green;
    --handle-background-checked: yellow;
}
```

**Other information:**

**Ionic info:**
```
Ionic:

   Ionic CLI                     : 5.2.4 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.7.4
   @angular-devkit/build-angular : 0.801.3
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.0.0

Utility:

   cordova-res : 0.5.2 (update available: 0.6.0)
   native-run  : not installed

System:

   NodeJS : v10.14.2 (/usr/local/bin/node)
   npm    : 6.10.2
   OS     : macOS Mojave```

**Comment by chrisgriffith** [comment]
Done. :)

**Comment by brandyscarney** [comment]
Thanks for the issue! This is a duplicate of https://github.com/ionic-team/ionic/issues/18940 which was closed by https://github.com/ionic-team/ionic/commit/94e525c10b688518f0e24b2b5d15ecbbde61678c.

This should be fixed in `4.8.0`: https://github.com/ionic-team/ionic/releases/tag/v4.8.0
